site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -235,7 +235,9 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.ping.udp.loadBalancerIP`                    | The [Load Balancer IP][loadBalancer] of the UDP Service load balancer. Only works if the Kubernetes provider supports this option.                | ""           |
 | `agones.ping.udp.loadBalancerSourceRanges`          | The [Load Balancer SourceRanges][loadBalancer] of the UDP Service load balancer. Only works if the Kubernetes provider supports this option.      | `[]`         |
 | `agones.allocator.disableMTLS`                      | Turns off client cert authentication for incoming connections to the allocator.            | `false`                |
-| `agones.allocator.disableTLS`                       | Turns off TLS security for incoming connections to the allocator. | `false`                |
+| `agones.allocator.disableTLS`                       | Turns off TLS security for incoming connections to the allocator.    | `false`                |
+| `agones.allocator.allocationTimeout`                | Remote allocation call timeout.                                      | `10s`                  |
```

Comment:
I am not sure if this is the right place to add it as it is inside the `feature publishVersion="1.9.0" ` if statement

Reply:
It should similarly go out with 1.10 release. The 1.9-RC is already merged.